provider/in_place_upgrade_base.py

```diff
@@ -130,6 +130,10 @@ def pre_upgrade_whitelist(self, test):
             error_info = self.params.get("error_info")
             if re.search(error_info, output):
                 pass
+            #rmmod unsupported modules since it's not supported on RHEL9
+            post_release = self.params.get("post_release")
+            if post_release == "release 9":
```

Comment:
As 9 is the psot release, so how about set this for RHEL8 in cfg
```
RHEL.8:
    rmmod_module = "rmmod ip_set;rmmod nft_compat"
```

Reply:
Agreed, I have updated it and could you please help to review it again ? @PaulYuuu

Reply:
Only need to check if `rmmod_module` is set, not need to check post release.

Reply:
Updated, thanks